Common/Utilities.cs

```diff
@@ -1,53 +1,47 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using CoreFtp;

```

Comment:
Does this sample use CoreFtp? If not remove the ref.

Reply:
Yes, this sample use CoreFtp in method UploadFileToWebApp(). Like i mentioned in other PR.